.github/workflows/pr-license.yaml

```diff
@@ -0,0 +1,19 @@
+name: Check License
```

Comment:
Change to `License Check` to be same format with other action name in this project.